string[] types
---
semap_msgs/ObjectInstance[] objects
